integration-tests/environments/react-native/package.json

```diff
@@ -116,6 +109,7 @@
     "test:ios": {
       "command": "npm run common",
       "dependencies": [
+        "../../../packages/realm:bindgen:generate:jsi",
```

Comment:
Shouldn't this be a dependency of the `pod-install` job instead?

Reply:
It would if this command generated the dummy files and inout list.  I think creating a script to do that would solve the problem